afs-storage-api/src/main/java/com/powsybl/afs/storage/check/FileSystemCheckIssue.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.storage.check;
+
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ */
+public class FileSystemCheckIssue {
+
+    private UUID uuid;
```

Comment:
The principle here was more to have an ID for the issue itself, since an issue could be not linked to a specific node.
But OK to leave it here as metadata !

No need to have an ID for the issue, since we decided to only select the TYPES of issues to be resolved, not the list of individual issues.

Reply:
Note: I have changed it to `String` because the AFS API does not enforce to have UUIDs.